### PR TITLE
style: add protocol audits

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Added protocol audits section to protocols page.

## Screenshots/Recordings

image

### CODE REVIEW

File: components/ProtocolPage/ProtocolAudits.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                OLAS, 23.06.22
+              </Link>
+            </li>
+            <li className="text-purple-600">
+              <Link
+                href="https://sourcehat.com/audits/AutonolasTokenomics/"
+                className="text-purple-600"
+              >
+                Tokenomics, 16.02.23
+              </Link>
```
Comment (truemiller) [inline components/ProtocolPage/ProtocolAudits.jsx:33]:
```suggestion
              <a
                href="https://sourcehat.com/audits/ValoryOLAS/"
                className="text-purple-600"
              >
                OLAS, 23.06.22
              </a>
            </li>
            <li className="text-purple-600">
              <a
                href="https://sourcehat.com/audits/AutonolasTokenomics/"
                className="text-purple-600"
              >
                Tokenomics, 16.02.23
              </a>
```

File: components/ProtocolPage/ProtocolAudits.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                  Tokenomics & Registries, 08.23
+                </Link>
+              </li>
+              <li className="text-purple-600">
+                <Link
+                  href="https://immunefi.com/bug-bounty/autonolas/information/"
+                  className="text-purple-600"
+                >
+                  Staking, 08.24
+                </Link>
```
Comment (truemiller) [inline components/ProtocolPage/ProtocolAudits.jsx:72]:
```suggestion
                <a
                  href="https://x.com/autonolas/status/1557431064632647680"
                  className="text-purple-600"
                  target="_blank"
                >
                  Governance & OLAS, 08.22
                </a>
              </li>
              <li>
                <a
                  href="https://x.com/autonolas/status/1689316293030359040"
                  className="text-purple-600"
                  target="_blank"
                >
                  Tokenomics & Registries, 08.23
                </a>
              </li>
              <li className="text-purple-600">
                <a
                  href="https://immunefi.com/bug-bounty/autonolas/information/"
                  className="text-purple-600"
                  target="_blank"
                >
                  Staking, 08.24
                </a>
```

File: components/ProtocolPage/ProtocolAudits.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        iconSrc: 'Cantina.png',
+        content: (
+          <ul className="list-disc ml-6 space-y-2">
+            <li>
+              <Link
+                href="https://x.com/valoryag/status/1769951643330810357"
+                className="text-purple-600"
+              >
+                Various contracts on Solana, 03.24
+              </Link>{' '}
```
Comment (truemiller) [inline components/ProtocolPage/ProtocolAudits.jsx:89]:
```suggestion
              <a
                href="https://x.com/valoryag/status/1769951643330810357"
                className="text-purple-600"
                target="_blank"
              >
                Various contracts on Solana, 03.24
              </a>{' '}
```

File: components/ProtocolPage/ProtocolAudits.jsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        content: (
+          <ul className="list-disc ml-6 space-y-2">
+            <li>
+              <Link
+                href="https://code4rena.com/audits/2023-12-olas"
+                className="text-purple-600"
+              >
+                Governance, Tokenomics & Registries on mainnet & contracts on
+                Solana, 12.23{' '}
+              </Link>{' '}
```
Comment (truemiller) [inline components/ProtocolPage/ProtocolAudits.jsx:107]:
```suggestion
              <a
                href="https://code4rena.com/audits/2023-12-olas"
                className="text-purple-600"
                target="_blank"
              >
                Governance, Tokenomics & Registries on mainnet & contracts on
                Solana, 12.23{' '}
              </a>{' '}
```

**Review by truemiller — DISMISSED** [review]
tiny; pls use `a` tags with `target=_blank` for external links

**Comment by contentwillvary** [comment]
@atepem great thanks, links and dates and details all correct afaict. please remove the . after ($50k). and ($90k). so it's consistent with the rest of the formatting, otherwise lgtm.

minor things that shouldn't block us: we don't need full line break between 'up to 50k' and 'governance' bullet, could be whatever line spacing is standard. could have a buffer spacing around the logos so they're not so close to headings above.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**